compress.go

```
@@ -162,6 +162,27 @@ func putFlateReader(fr io.Reader) {
 	flateReaderPool.Put(fr)
 }
 
+var flateWriterPool = [11]sync.Pool{}
```

golangci-lint (2.12.2)

[error] var flateWriterPool is unused

(unused)